ironfish/src/webApi.ts

```diff
@@ -93,16 +93,16 @@ export class WebApi {
     return response.data
   }
 
-  async getNextFaucetTransaction(): Promise<FaucetTransaction | null> {
+  async getNextFaucetTransactions(count: number): Promise<FaucetTransaction[] | null> {
     this.requireToken()
 
     try {
-      const response = await axios.get<FaucetTransaction>(
-        `${this.host}/faucet_transactions/next`,
+      const response = await axios.get<{ object: string; data: FaucetTransaction[] }>(
```

Comment:
what is "object" here? This is a pretty weird response type from the API. What is it sending back?

Reply:
The API is using the `List` type (which includes an object field [usually `list` in this case] and a data field) on this endpoint for consistency with other endpoints that return a list.

Reply:
I mean that's fine, but I don't understand the point of the object field that's typed as a generic "string". What is the value? When can it not be list? Why is it returned to the front end in the first place? It's especially not even used here, so who would use it?

Reply:
Object field has been removed with no harm to functionality. 👍🏽